app/serializers/v1/invoice_serializer.rb

```diff
@@ -64,7 +64,7 @@ def metadata
     end
 
     def applied_taxes
-      ::CollectionSerializer.new(model.applied_taxes, ::V1::TaxSerializer, collection_name: 'applied_taxes').serialize
+      ::CollectionSerializer.new(model.applied_taxes, ::V1::Invoices::AppliedTaxSerializer, collection_name: 'applied_taxes').serialize
```

Comment:
Layout/LineLength: Line is too long. [135/120] (https://rubystyle.guide#max-line-length)